File: examples/javascript/constructs/for_loop.js
// Ported from examples/lumen/constructs/for_loop.lm by scripts/port_examples.py; edit the Lumen original, not this file.
let i = 0;
while (i < 10) {
    process.stdout.write(String(i));
    if (i < 9) {
        process.stdout.write(", ");
    }
    i = i + 1;
}
console.log("");

File: examples/rust/constructs/for_loop.rs
// Ported from examples/lumen/constructs/for_loop.lm by scripts/port_examples.py; edit the Lumen original, not this file.
fn main() {
    for i in 0..10 {
        print!("{}", i);
        if i < 9 {
            print!(", ");
        }
    }
    println!("");
}
